configserv/image.yaml

```diff
@@ -1,4 +1,4 @@
-version: 1.0
+version: latest
```

Comment:
Ideally we'd like to parameterize this, as in a CI environment, we'd like to continuously build 'latest', but use a fixed version for tags and/or branches.